src/components/rounds-list/rounds-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Round } from "@/app/events/[event]/participants/page";
+import { ParticipantsList } from "../participants-list";
+
+interface RoundsListProps {
+  rounds: Round[];
+}
+
+export function RoundsList({ rounds }: RoundsListProps) {
+  return (
+    <div className="inline-block">
+      <div className="inline-flex flex-wrap md:gap-x-8 md:gap-y-10 max-w-[1000px]">
+        {rounds.map((round, index) => (
+          <div className="mt-8 md:mt-0" key={index}>
+            <span className="text-neutral-500 text-sm font-semibold leading-tight block">
+              {index + 1}º Round
+            </span>
+            <ParticipantsList participants={round.participants} />
```

Comment:
This component could be more flexible to avoid the usage of bail out of elements. My suggestion: Let's break it into four components `ParticipantsList`, `ParticipantItem`, `ParticipantImage` and `ParticipantName`.
```suggestion
<ParticipantsList>
  {participants.map((participant) => (
    <ParticipantItem>
     <PaticipantImage src={participant.avatarUrl} />
     <ParticipantName>{participant.name></Participant>
    </ParticipantItem>
  ))
</ParticipantsList>

Reply:
Done!